app/uk/gov/hmrc/alcoholdutyaccount/config/AppConfig.scala

```diff
@@ -63,7 +63,8 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
   def minimumHistoricPaymentsYear: Int =
     config.get[Int]("payments.minimumHistoricPaymentsYear")
 
-  def btaServiceAvailable: Boolean = config.get[Boolean]("features.bta-service-available")
+  val ofpSubscriptionAvailable: Boolean = config.get[Boolean]("features.ofp-subscription-available")
```

Comment:
I'd prefer something like ofpSubscriptionEnabled here to make it more clear that this is a feature toggle

Reply:
That's a reasonable point. I'll ask Yash about his preference for the name.